crates/bevy_window/src/window.rs

```diff
@@ -191,6 +193,7 @@ impl Default for Window {
     fn default() -> Self {
         Self {
             title: "Bevy App".to_owned(),
+            app_id: "bevy.app".to_owned(),
```

Comment:
is it an issue if several apps running at the same time have the same `app_id`? Would it be better as an option, and only set it when the user actually put a value?

Reply:
drive-by comment: It's not semantically invalid, but it's not useful to have a bunch of unrelated programs with the same `app_id`. So I'd say that it should be an `Option`.

Reply:
I made this an `Option<String>` type and `None` by default now.